cmake/Package.cmake

```diff
@@ -0,0 +1,24 @@
+# -- packaging -----------------------------------------------------------------
+
+set(CPACK_PACKAGING_INSTALL_PREFIX "/usr")
```

Comment:
Shouldn't this respect the CMAKE_INSTALL_PREFIX?

Reply:
Yeah I think we can just remove that.